### ISSUE TITLE
TM1py is missing a pyproject.toml installer

### ISSUE THREAD

**Issue by zallin93** [cubewise-code/tm1py#824]
**Describe the bug**
During one of our most recent deployments, we started receiving deprecation notices from PIP that TM1py and some of its peer dependencies are missing the required `pyproject.toml` setup file. 

```  
DEPRECATION: mdxpy is being installed using the legacy 'setup.py install' method, because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. pip 23.1 will enforce this behaviour change. A possible replacement is to enable the '--use-pep517' option. Discussion can be found at https://github.com/pypa/pip/issues/8559
Running setup.py install for mdxpy ... done
DEPRECATION: TM1py is being installed using the legacy 'setup.py install' method, because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. pip 23.1 will enforce this behaviour change. A possible replacement is to enable the '--use-pep517' option. Discussion can be found at https://github.com/pypa/pip/issues/8559
Running setup.py install for TM1py ... done
```

In addition to a `setup.py` configuration, TM1py should supply a `pyproject.toml` install config. 

**To Reproduce**
Install TM1py using Pip 

**Expected behavior**
TM1py should install without deprecation warnings.

**Version**
TM1py: latest, 1.10.1
Python 3.9.6
Pip: 22.3.1

**Comment by zallin93** [comment]
Related: https://github.com/pypa/pip/issues/8559

**Comment by scrambldchannel** [comment]
I've added a quick fix in a PR. From the issue @zallin93 links to, it reads like you have until April next year or so.

**Comment by MariusWirtz** [comment]
Solved with #826

To be released before the end of the year with 1.10.2
